.github/workflows/centaur_integration_tests.yml

```diff
@@ -0,0 +1,61 @@
+name: 'Centaur Integration Tests'
+
+#This github action runs all of Cromwell's integration tests.
+
+#This is what shows up in the github workflows page as the title. 
+run-name: ${{ github.actor }} Centaur Testing.
+
+#What will trigger the workflow to run. 
+on:
+  workflow_dispatch: #Manual trigger from GitHub UI
+  push: #git push
+
+permissions: 
+  contents: read
+
+jobs:
+  centaur-tests:
+    strategy:
+      fail-fast: false #disabling fail-fast means that even if one test fails, the others will still try to complete.
+      #Each entry below is a single integration test that lives in /src/ci/bin/. 
+      #Each will be launched on its own runner so they can occur in parallel.
+      matrix:
+        include:
+          - build_type: centaurPapiV2beta
+            build_mysql: 5.7
```

Comment:
It looks like this 5.7 gets included in the test name we see on the main PR page, but nothing makes it clear that it's describing mysql version. Is there an easy way to include that label?

Reply:
I added a "Friendly Name" field we can use for each test so we can see the more specific info from the Github UI :) 

The more elegant way of automagically using these include variables in the UI name isn't trivial at the moment because we're relying on multiple, different variables for the different databases we use.